consoles/video_stream.pm

```diff
@@ -225,6 +228,28 @@ sub send_key_event ($self, $key, $press_release_delay) {
       or die "failed to send '$key' input event";
 }
 
+sub type_string ($self, $args) {
+    my $keyboard_device_url = $bmwqemu::vars{GENERAL_HW_PICO_KEYBOARD_URL};
+    if ($keyboard_device_url) {
+        # send keyboard events using RPi Pico W based keyboard emulator
+        my $url = Mojo::URL->new($keyboard_device_url)->query(type => $args->{text});
+        bmwqemu::diag(Mojo::UserAgent->new->get($url)->result->body);
```

Comment:
This diag message is useless without better context information. In this form it'll just make a URL show up in the logs with no hint where it comes from and why it was logged. Either remove the logging completely or provide better context. Note that printing the whole body unconditionally as-is might also be problematic (as an HTTP request might generally return all kinds of data, e.g. possibly binary data).

Reply:
I don't print the url but the body.
This device will only return "text/plain" on this api method:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/5f476954266b55e998acab3ce532aaddf4cc44d4/data/generalhw_scripts/rpi_pico_w_keyboard/rpi_pico_w_keyboard.ino#L189-L209

Reply:
But it still lacks context. So at least add a prefix like `response from typing string via HTTP request: `. Then I'll at least be clear where the trash being logged here comes from. And yes, we must assume that the HTTP server possibly returns trash, even though your C++ code is of course bullet-proof :-)

Reply:
added some context